### DISCUSSION TITLE
Dynamic Power Limiter - Startet teilweise für ein paar Sekunden den Inverter obwohl Startspannung nicht erreicht

### DISCUSSION THREAD

**Discussion by peff74** [hoylabs/OpenDTU-OnBattery#206] [Q&A] [answered]
Moin,

habe ich da ein Verständnisproblem, oder wie kommt es zustande?
Einstellungen sind wie folgt:

![2023-04-29 09_01_12-OpenDTU](https://user-images.githubusercontent.com/91614910/235288755-0c42f5cd-795c-4b9a-94de-29d38863fecf.png)

Er hat nun bei ~39.7V abgeschaltet. Aber ab und zu wird der Wechselrichter eingeschaltet und Produziert?!?
Ist mir gestern Abend schon im Webinterface aufgefallen und heute Morgen wo MQTT nun geht, sehe ich es in den Graphen. 

![2023-04-29 09_04_20-Program Manager](https://user-images.githubusercontent.com/91614910/235288898-59fff09e-c807-4981-bbb6-661dba96f5df.png)

Irgendeine Idee?
Gruß
Peff

**Comment by helgeerbe** [comment, marked as answer]
Ja, ist ein Bug. Wir haben den PowerLimiter neu geschrieben, damit der Code wartbarer wird.  Versuche heute eine neue Release zu bauen.
